d3/d3.d.ts

```diff
@@ -119,7 +119,7 @@ declare module D3 {
         * @param arr Array to search
         * @param map Accsessor function
         */
-        extent<T, U>(arr: T[], map: (v: T) => U): U[];
+        extent<T, U>(arr: T[], map: (v: T, i?: number) => U): U[];
```

Comment:
It should, since you should be able to place a function that statically returns a constant

function toThrees () { return 3; } should work in an array.map